bin/wmagent-component-standalone

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env python
+"""
+wmagent-component-standalone
+
+Utility script for running a WMAgent component standalone.
+"""
+
+import os
+import sys
+import logging
+import importlib
+
+from argparse import ArgumentParser
+from WMCore.Configuration import loadConfigurationFile
+from WMCore.ResourceControl.ResourceControl import ResourceControl
+from WMCore.Services.CRIC.CRIC import CRIC
+from WMCore.WMInit import connectToDB
+
+
+def createOptionParser():
+    """
+    _createOptionParser_
+
+    Creates an option parser to setup the component run parameters
+    """
+    optParser = ArgumentParser()
+    optParser.add_argument("--config", dest="wmaConfigPath", help="WMAgent configuration file.",
+                           default=os.environ.get("WMAGENT_CONFIG", None))
+    optParser.add_argument("--daemon", dest="daemon",
```

Comment:
I don't see this option being used in the script, other than for a print. Is it supposed to be a placeholder?

Reply:
it will be... within one of the next commits to come.